cpu/stm32f1/cpu.c

```diff
@@ -95,19 +95,25 @@
 #endif
 #endif
 
+#ifndef FW_SLOTS
```

Comment:
I don't like this...  AFAIK the clock can be initialized several times but there might be registers which might need to be set to the "default" value. 

**EDIT**: maybe a different initialization cpu is cleaner. Maybe for the bootloader only the internal oscillator and not all peripherals are needed.